modules/virt-retrieving-vm-definition-from-vme-manifest.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-exporting-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-retrieving-vm-definition-from-vme-manifest_{context}"]
+= Retrieving a VM definition from a virtual machine export manifest
+
+After you export a virtual machine (VM) or snapshot, you can access the `VirtualMachine` manifest and related information from the export server.
+
+.Prerequisites
+
+* You exported a virtual machine or VM snapshot by creating a `VirtualMachineExport` custom resource (CR).
++
+[NOTE]
+====
+`VirtualMachineExport` objects that have `spec.source.kind: PersistentVolumeClaim` do not generate virtual machine manifests.
+====
+
+.Procedure
+
+. Get the `VirtualMachineExport` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc get vmexport <export_name> -o yaml <1>
+----
+<1> Replace `<export_name>` with the `metadata.name` value from the `VirtualMachineExport` object.
+
+. Review the `status.links` stanza, which is divided into `external` and `internal` sections. Note the `manifests.url` fields within each section:
++
+.Example output
+
+[source,yaml]
+----
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: example-export
+spec:
+  source:
+    apiGroup: "kubevirt.io"
+    kind: VirtualMachine
+    name: example-vm
+  tokenSecretRef: example-token
+status:
+...
+  links:
+    external:
+...
+      manifests:
+      - type: all
+        url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/all <1>
+      - type: auth-header-secret
+        url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/secret <2>
+    internal:
+...
+      manifests:
+      - type: all
+        url: https://virt-export-export-pvc.default.svc/internal/manifests/all <3>
+      - type: auth-header-secret
+        url: https://virt-export-export-pvc.default.svc/internal/manifests/secret
+  phase: Ready
+  serviceName: virt-export-example-export
+----
+<1> Access the `VirtualMachine` manifest, `DataVolume` objects, if present, and a config map that contains the public certificate for the external URL's ingress or route.
+<2> Access a secret containing a header that is compatible with Containerized Data Importer (CDI). The header contains a text version of the export token.
+<3> Access the `VirtualMachine` manifest, `DataVolume` objects, if present, and a config map that contains the certificate authority (CA) of the internal URL's export server.
```

Comment:
```suggestion
<3> Access the `VirtualMachine` manifest, `DataVolume` manifest, if present, and a `ConfigMap` that contains the certificate for the internal URL's export server.
```

Reply:
Changed "objects" to "manifest" in both locations, but I changed "config map" to "`ConfigMap` manifest" to follow our API object formatting guidelines ([link if you're curious](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)). Let me know if this works.